pkg/kube/kube.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package kube
```

Comment:
I feel like this needs a more obvious package name. Is `kube` kubernetes or `minikube`, 'kubeadm'? It feels like it could be so many things. These are all calls that talk to the Kubernetes API server. 

Some alternative ideas: `kubernetes`, `kapi`, `kubeclient`, 'apiclient', 'kapiclient'?

Reply:
I like kapi ! the kubeclient would stutter for the name kbueclient.Client()